pkg/config/app/kuma-dp/config.go

```diff
@@ -188,19 +189,30 @@ func (d *Dataplane) Sanitize() {
 
 func (d *Dataplane) Validate() (errs error) {
 	proxyType := mesh_proto.ProxyType(d.ProxyType)
-	if err := proxyType.IsValid(); err != nil {
-		errs = multierr.Append(errs, errors.Wrap(err, ".ProxyType is not valid"))
+	switch proxyType {
+	case v1alpha1.DataplaneProxyType, v1alpha1.IngressProxyType:
```

Comment:
We're trying to avoid `v1alpha1` and use named imports like `mesh_proto` for example. I guess `mesh_proto` name already present in this file, so I guess we have to use it instead of `v1alpha`

Reply:
Ugh, very sloppy on my part! Good catch, fixed :)